pkg/virt-operator/resource/generate/components/prometheus.go

```diff
@@ -307,7 +307,7 @@ func NewPrometheusRuleSpec(ns string, workloadUpdatesEnabled bool) *v1.Prometheu
 					},
 					{
 						Alert: "NoReadyVirtOperator",
-						Expr:  intstr.FromString("kubevirt_virt_operator_ready_total == 0"),
+						Expr:  intstr.FromString("absent(kubevirt_virt_operator_ready_total) or kubevirt_virt_operator_ready_total == 0"),
```

Comment:
https://github.com/kubevirt/kubevirt/pull/7529 fixes these issues.

Reply:
changes removed

Reply:
Thank you!